sense/display.py

```diff
@@ -181,8 +191,75 @@ def display(self, img: np.ndarray, display_data: dict) -> np.ndarray:
             text_color = self.default_text_color
 
         # Show FPS on the video screen
-        put_text(img, "Camera FPS: {:.1f}".format(camera_fps), (5, img.shape[0] - self.y_offset - 20), text_color)
-        put_text(img, "Model FPS: {:.1f}".format(inference_engine_fps), (5, img.shape[0] - self.y_offset), text_color)
+        put_text(img, "Camera FPS: {:.1f}".format(camera_fps), (5, img.shape[0] - self.y_offset - 20),
+                 color=text_color)
+        put_text(img, "Model FPS: {:.1f}".format(inference_engine_fps), (5, img.shape[0] - self.y_offset),
+                 color=text_color)
+
+        return img
+
+
+class DisplayDigits(BaseDisplay):
```

Comment:
You are using this for digits, but it could be used for any recognized class name, right? Maybe rename to something more generic (`DisplayClassnameOverlay` or similar)?
Then you can maybe reduce the default font scale so that a typical class name (10 to 20 characters) can be displayed nicely and only your example script sets the larger display parameters.

Reply:
Good idea :slightly_smiling_face: